logs
DEBUG:tweepy.api:PARAMS: {}
DEBUG:requests_oauthlib.oauth1_auth:Signing request <PreparedRequest [GET]> using client <Client client_key=HQcZWdKO9rVfRocK5nhGX86eG, client_secret=****, resource_owner_key=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX, resource_owner_secret=****, signature_method=HMAC-SHA1, signature_type=AUTH_HEADER, callback_uri=None, rsa_key=None, verifier=None, realm=None, encoding=utf-8, decoding=None, nonce=None, timestamp=None>
DEBUG:requests_oauthlib.oauth1_auth:Including body in call to sign: False
DEBUG:oauthlib.oauth1.rfc5849:Collected params: [('oauth_nonce', '82512127959491308391681884170'), ('oauth_timestamp', '1681884170'), ('oauth_version', '1.0'), ('oauth_signature_method', 'HMAC-SHA1'), ('oauth_consumer_key', 'HQcZWdKO9rVfRocK5nhGX86eG'), ('oauth_token', '1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX')]
DEBUG:oauthlib.oauth1.rfc5849:Normalized params: oauth_consumer_key=HQcZWdKO9rVfRocK5nhGX86eG&oauth_nonce=82512127959491308391681884170&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1681884170&oauth_token=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX&oauth_version=1.0
DEBUG:oauthlib.oauth1.rfc5849:Normalized URI: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:oauthlib.oauth1.rfc5849:Signing: signature base string: GET&https%3A%2F%2Fapi.twitter.com%2F1.1%2Faccount%2Fverify_credentials.json&oauth_consumer_key%3DHQcZWdKO9rVfRocK5nhGX86eG%26oauth_nonce%3D82512127959491308391681884170%26oauth_signature_method%3DHMAC-SHA1%26oauth_timestamp%3D1681884170%26oauth_token%3D1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX%26oauth_version%3D1.0
DEBUG:oauthlib.oauth1.rfc5849:Signature: 0Iv6bzUkMzYHbzytKsK7yQDFpuE=
DEBUG:requests_oauthlib.oauth1_auth:Updated url: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:requests_oauthlib.oauth1_auth:Updated headers: {'User-Agent': 'Python/3.8.16 Requests/2.28.2 Tweepy/4.13.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Authorization': 'OAuth oauth_nonce="82512127959491308391681884170", oauth_timestamp="1681884170", oauth_version="1.0", oauth_signature_method="HMAC-SHA1", oauth_consumer_key="HQcZWdKO9rVfRocK5nhGX86eG", oauth_token="1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX", oauth_signature="0Iv6bzUkMzYHbzytKsK7yQDFpuE%3D"'}
DEBUG:requests_oauthlib.oauth1_auth:Updated body: None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.twitter.com:443
DEBUG:urllib3.connectionpool:https://api.twitter.com:443 "GET /1.1/account/verify_credentials.json HTTP/1.1" 200 565
INFO:root:Conexión a Twitter Exitosa
DEBUG:asyncio:Using selector: EpollSelector
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.56ms
ERROR:tornado.application:Uncaught exception POST /tweets (172.89.0.1)
HTTPServerRequest(protocol='http', host='localhost:5071', method='POST', uri='/tweets', version='HTTP/1.1', remote_ip='172.89.0.1')
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/tornado/web.py", line 1702, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "backend_main.py", line 78, in post
    respuesta = tweets()
  File "backend_main.py", line 172, in tweets
    count = tweet['count']
KeyError: 'count'
ERROR:tornado.access:500 POST /tweets (172.89.0.1) 38.52ms
DEBUG:tweepy.api:PARAMS: {}
DEBUG:requests_oauthlib.oauth1_auth:Signing request <PreparedRequest [GET]> using client <Client client_key=HQcZWdKO9rVfRocK5nhGX86eG, client_secret=****, resource_owner_key=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX, resource_owner_secret=****, signature_method=HMAC-SHA1, signature_type=AUTH_HEADER, callback_uri=None, rsa_key=None, verifier=None, realm=None, encoding=utf-8, decoding=None, nonce=None, timestamp=None>
DEBUG:requests_oauthlib.oauth1_auth:Including body in call to sign: False
DEBUG:oauthlib.oauth1.rfc5849:Collected params: [('oauth_nonce', '11098921875564717741681884435'), ('oauth_timestamp', '1681884435'), ('oauth_version', '1.0'), ('oauth_signature_method', 'HMAC-SHA1'), ('oauth_consumer_key', 'HQcZWdKO9rVfRocK5nhGX86eG'), ('oauth_token', '1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX')]
DEBUG:oauthlib.oauth1.rfc5849:Normalized params: oauth_consumer_key=HQcZWdKO9rVfRocK5nhGX86eG&oauth_nonce=11098921875564717741681884435&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1681884435&oauth_token=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX&oauth_version=1.0
DEBUG:oauthlib.oauth1.rfc5849:Normalized URI: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:oauthlib.oauth1.rfc5849:Signing: signature base string: GET&https%3A%2F%2Fapi.twitter.com%2F1.1%2Faccount%2Fverify_credentials.json&oauth_consumer_key%3DHQcZWdKO9rVfRocK5nhGX86eG%26oauth_nonce%3D11098921875564717741681884435%26oauth_signature_method%3DHMAC-SHA1%26oauth_timestamp%3D1681884435%26oauth_token%3D1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX%26oauth_version%3D1.0
DEBUG:oauthlib.oauth1.rfc5849:Signature: VlnrZj0sFYuIrvt12op0Pk3AS4U=
DEBUG:requests_oauthlib.oauth1_auth:Updated url: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:requests_oauthlib.oauth1_auth:Updated headers: {'User-Agent': 'Python/3.8.16 Requests/2.28.2 Tweepy/4.13.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Authorization': 'OAuth oauth_nonce="11098921875564717741681884435", oauth_timestamp="1681884435", oauth_version="1.0", oauth_signature_method="HMAC-SHA1", oauth_consumer_key="HQcZWdKO9rVfRocK5nhGX86eG", oauth_token="1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX", oauth_signature="VlnrZj0sFYuIrvt12op0Pk3AS4U%3D"'}
DEBUG:requests_oauthlib.oauth1_auth:Updated body: None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.twitter.com:443
DEBUG:urllib3.connectionpool:https://api.twitter.com:443 "GET /1.1/account/verify_credentials.json HTTP/1.1" 200 565
INFO:root:Conexión a Twitter Exitosa
DEBUG:asyncio:Using selector: EpollSelector
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.55ms
INFO:tornado.access:200 POST /tweets (172.89.0.1) 33.50ms
INFO:tornado.access:200 OPTIONS /hashtags (172.89.0.1) 0.53ms
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.34ms
INFO:tornado.access:200 POST /hashtags (172.89.0.1) 11.03ms
INFO:tornado.access:200 POST /tweets (172.89.0.1) 31.91ms
INFO:tornado.access:200 OPTIONS /hashtags (172.89.0.1) 0.42ms
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.57ms
INFO:tornado.access:200 POST /hashtags (172.89.0.1) 12.57ms
INFO:tornado.access:200 POST /tweets (172.89.0.1) 34.85ms
DEBUG:tweepy.api:PARAMS: {}
DEBUG:requests_oauthlib.oauth1_auth:Signing request <PreparedRequest [GET]> using client <Client client_key=HQcZWdKO9rVfRocK5nhGX86eG, client_secret=****, resource_owner_key=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX, resource_owner_secret=****, signature_method=HMAC-SHA1, signature_type=AUTH_HEADER, callback_uri=None, rsa_key=None, verifier=None, realm=None, encoding=utf-8, decoding=None, nonce=None, timestamp=None>
DEBUG:requests_oauthlib.oauth1_auth:Including body in call to sign: False
DEBUG:oauthlib.oauth1.rfc5849:Collected params: [('oauth_nonce', '94228969473156838731681884639'), ('oauth_timestamp', '1681884639'), ('oauth_version', '1.0'), ('oauth_signature_method', 'HMAC-SHA1'), ('oauth_consumer_key', 'HQcZWdKO9rVfRocK5nhGX86eG'), ('oauth_token', '1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX')]
DEBUG:oauthlib.oauth1.rfc5849:Normalized params: oauth_consumer_key=HQcZWdKO9rVfRocK5nhGX86eG&oauth_nonce=94228969473156838731681884639&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1681884639&oauth_token=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX&oauth_version=1.0
DEBUG:oauthlib.oauth1.rfc5849:Normalized URI: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:oauthlib.oauth1.rfc5849:Signing: signature base string: GET&https%3A%2F%2Fapi.twitter.com%2F1.1%2Faccount%2Fverify_credentials.json&oauth_consumer_key%3DHQcZWdKO9rVfRocK5nhGX86eG%26oauth_nonce%3D94228969473156838731681884639%26oauth_signature_method%3DHMAC-SHA1%26oauth_timestamp%3D1681884639%26oauth_token%3D1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX%26oauth_version%3D1.0
DEBUG:oauthlib.oauth1.rfc5849:Signature: d0cpAT84oQ8J4MTkPhZVhSWoD8U=
DEBUG:requests_oauthlib.oauth1_auth:Updated url: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:requests_oauthlib.oauth1_auth:Updated headers: {'User-Agent': 'Python/3.8.16 Requests/2.28.2 Tweepy/4.13.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Authorization': 'OAuth oauth_nonce="94228969473156838731681884639", oauth_timestamp="1681884639", oauth_version="1.0", oauth_signature_method="HMAC-SHA1", oauth_consumer_key="HQcZWdKO9rVfRocK5nhGX86eG", oauth_token="1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX", oauth_signature="d0cpAT84oQ8J4MTkPhZVhSWoD8U%3D"'}
DEBUG:requests_oauthlib.oauth1_auth:Updated body: None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.twitter.com:443
DEBUG:urllib3.connectionpool:https://api.twitter.com:443 "GET /1.1/account/verify_credentials.json HTTP/1.1" 200 565
INFO:root:Conexión a Twitter Exitosa
DEBUG:asyncio:Using selector: EpollSelector
INFO:tornado.access:200 OPTIONS /hashtags (172.89.0.1) 0.48ms
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.39ms
INFO:tornado.access:200 POST /hashtags (172.89.0.1) 18.57ms
INFO:tornado.access:200 POST /tweets (172.89.0.1) 32.42ms
INFO:tornado.access:200 OPTIONS /hashtags (172.89.0.1) 0.43ms
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.32ms
INFO:tornado.access:200 POST /hashtags (172.89.0.1) 13.50ms
INFO:tornado.access:200 POST /tweets (172.89.0.1) 30.05ms
INFO:tornado.access:200 OPTIONS /hashtags (172.89.0.1) 0.45ms
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.29ms
INFO:tornado.access:200 POST /hashtags (172.89.0.1) 12.62ms
INFO:tornado.access:200 POST /tweets (172.89.0.1) 33.12ms
DEBUG:tweepy.api:PARAMS: {}
DEBUG:requests_oauthlib.oauth1_auth:Signing request <PreparedRequest [GET]> using client <Client client_key=HQcZWdKO9rVfRocK5nhGX86eG, client_secret=****, resource_owner_key=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX, resource_owner_secret=****, signature_method=HMAC-SHA1, signature_type=AUTH_HEADER, callback_uri=None, rsa_key=None, verifier=None, realm=None, encoding=utf-8, decoding=None, nonce=None, timestamp=None>
DEBUG:requests_oauthlib.oauth1_auth:Including body in call to sign: False
DEBUG:oauthlib.oauth1.rfc5849:Collected params: [('oauth_nonce', '97244454984557360451681885787'), ('oauth_timestamp', '1681885787'), ('oauth_version', '1.0'), ('oauth_signature_method', 'HMAC-SHA1'), ('oauth_consumer_key', 'HQcZWdKO9rVfRocK5nhGX86eG'), ('oauth_token', '1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX')]
DEBUG:oauthlib.oauth1.rfc5849:Normalized params: oauth_consumer_key=HQcZWdKO9rVfRocK5nhGX86eG&oauth_nonce=97244454984557360451681885787&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1681885787&oauth_token=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX&oauth_version=1.0
DEBUG:oauthlib.oauth1.rfc5849:Normalized URI: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:oauthlib.oauth1.rfc5849:Signing: signature base string: GET&https%3A%2F%2Fapi.twitter.com%2F1.1%2Faccount%2Fverify_credentials.json&oauth_consumer_key%3DHQcZWdKO9rVfRocK5nhGX86eG%26oauth_nonce%3D97244454984557360451681885787%26oauth_signature_method%3DHMAC-SHA1%26oauth_timestamp%3D1681885787%26oauth_token%3D1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX%26oauth_version%3D1.0
DEBUG:oauthlib.oauth1.rfc5849:Signature: g/AsfCVnw0Pkp2j4vj+rshBvmQE=
DEBUG:requests_oauthlib.oauth1_auth:Updated url: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:requests_oauthlib.oauth1_auth:Updated headers: {'User-Agent': 'Python/3.8.16 Requests/2.28.2 Tweepy/4.13.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Authorization': 'OAuth oauth_nonce="97244454984557360451681885787", oauth_timestamp="1681885787", oauth_version="1.0", oauth_signature_method="HMAC-SHA1", oauth_consumer_key="HQcZWdKO9rVfRocK5nhGX86eG", oauth_token="1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX", oauth_signature="g%2FAsfCVnw0Pkp2j4vj%2BrshBvmQE%3D"'}
DEBUG:requests_oauthlib.oauth1_auth:Updated body: None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.twitter.com:443
DEBUG:urllib3.connectionpool:https://api.twitter.com:443 "GET /1.1/account/verify_credentials.json HTTP/1.1" 200 565
INFO:root:Conexión a Twitter Exitosa
DEBUG:asyncio:Using selector: EpollSelector
INFO:tornado.access:200 OPTIONS /hashtags (172.89.0.1) 0.82ms
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.95ms
INFO:tornado.access:200 POST /hashtags (172.89.0.1) 19.49ms
INFO:tornado.access:200 POST /tweets (172.89.0.1) 35.38ms
DEBUG:tweepy.api:PARAMS: {}
DEBUG:requests_oauthlib.oauth1_auth:Signing request <PreparedRequest [GET]> using client <Client client_key=HQcZWdKO9rVfRocK5nhGX86eG, client_secret=****, resource_owner_key=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX, resource_owner_secret=****, signature_method=HMAC-SHA1, signature_type=AUTH_HEADER, callback_uri=None, rsa_key=None, verifier=None, realm=None, encoding=utf-8, decoding=None, nonce=None, timestamp=None>
DEBUG:requests_oauthlib.oauth1_auth:Including body in call to sign: False
DEBUG:oauthlib.oauth1.rfc5849:Collected params: [('oauth_nonce', '19820155108019826981681885954'), ('oauth_timestamp', '1681885954'), ('oauth_version', '1.0'), ('oauth_signature_method', 'HMAC-SHA1'), ('oauth_consumer_key', 'HQcZWdKO9rVfRocK5nhGX86eG'), ('oauth_token', '1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX')]
DEBUG:oauthlib.oauth1.rfc5849:Normalized params: oauth_consumer_key=HQcZWdKO9rVfRocK5nhGX86eG&oauth_nonce=19820155108019826981681885954&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1681885954&oauth_token=1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX&oauth_version=1.0
DEBUG:oauthlib.oauth1.rfc5849:Normalized URI: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:oauthlib.oauth1.rfc5849:Signing: signature base string: GET&https%3A%2F%2Fapi.twitter.com%2F1.1%2Faccount%2Fverify_credentials.json&oauth_consumer_key%3DHQcZWdKO9rVfRocK5nhGX86eG%26oauth_nonce%3D19820155108019826981681885954%26oauth_signature_method%3DHMAC-SHA1%26oauth_timestamp%3D1681885954%26oauth_token%3D1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX%26oauth_version%3D1.0
DEBUG:oauthlib.oauth1.rfc5849:Signature: /uSz2H7PUIXdC9ToHkbiItOG11Y=
DEBUG:requests_oauthlib.oauth1_auth:Updated url: https://api.twitter.com/1.1/account/verify_credentials.json
DEBUG:requests_oauthlib.oauth1_auth:Updated headers: {'User-Agent': 'Python/3.8.16 Requests/2.28.2 Tweepy/4.13.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Authorization': 'OAuth oauth_nonce="19820155108019826981681885954", oauth_timestamp="1681885954", oauth_version="1.0", oauth_signature_method="HMAC-SHA1", oauth_consumer_key="HQcZWdKO9rVfRocK5nhGX86eG", oauth_token="1646478811620429825-o8y3bqQeE5ySEUA29KyeNUnFO99xyX", oauth_signature="%2FuSz2H7PUIXdC9ToHkbiItOG11Y%3D"'}
DEBUG:requests_oauthlib.oauth1_auth:Updated body: None
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.twitter.com:443
DEBUG:urllib3.connectionpool:https://api.twitter.com:443 "GET /1.1/account/verify_credentials.json HTTP/1.1" 200 565
INFO:root:Conexión a Twitter Exitosa
DEBUG:asyncio:Using selector: EpollSelector
INFO:tornado.access:200 OPTIONS /hashtags (172.89.0.1) 0.36ms
INFO:tornado.access:200 OPTIONS /tweets (172.89.0.1) 0.31ms
INFO:tornado.access:200 POST /hashtags (172.89.0.1) 20.40ms
ERROR:tornado.application:Uncaught exception POST /tweets (172.89.0.1)
HTTPServerRequest(protocol='http', host='localhost:5071', method='POST', uri='/tweets', version='HTTP/1.1', remote_ip='172.89.0.1')
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/tornado/web.py", line 1702, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "backend_main.py", line 79, in post
    respuesta = tweets()
  File "backend_main.py", line 178, in tweets
    paises = list(set([tweet["pais"].upper() for tweet in list_tweets]))
  File "backend_main.py", line 178, in <listcomp>
    paises = list(set([tweet["pais"].upper() for tweet in list_tweets]))
AttributeError: 'NoneType' object has no attribute 'upper'
ERROR:tornado.access:500 POST /tweets (172.89.0.1) 26.55ms
